crates/hc_sandbox/tests/cli.rs

```diff
@@ -71,10 +71,29 @@ async fn get_app_info(admin_port: u16, installed_app_id: InstalledAppId, port: u
         .await
         .unwrap();
 
-    let request = AppRequest::AppInfo;
-    let response = app_tx.request(request);
-    let r: AppResponse = check_timeout(response).await;
-    assert_matches!(r, AppResponse::AppInfo(Some(_)));
+    tokio::time::timeout(Duration::from_secs(60), async move {
```

Comment:
Hmm...by moving this into a timeout here, would it still be possible to have the `AppResonse` be returned from the `get_app_info()` function? This would be useful for [this](https://github.com/holochain/holochain/blob/508959761af0b773cbce987f89fed6c9aa25e172/crates/hc_sandbox/tests/cli.rs#L311-L318) case of the pending PR to add custom properties to hc sandbox.

Reply:
The value can still be returned